gcc/testsuite/gcc.target/riscv/rvv/vtype-call-clobbered.c

```diff
@@ -1,5 +1,5 @@
 /* { dg-do compile } */
-/* { dg-options "-march=rv64gcv -mabi=lp64 -O2" } */
+/* { dg-options "-march=rv64gcv -mabi=lp64d -O2" } */
```

Comment:
If this is unrelated to the current PR, I would remove it.

Reply:
I don't even know where this change comes from. O.o